src/cpu/gemm/gemm.cpp

```diff
@@ -30,6 +30,10 @@
 #include "cpu/gemm/f32/ref_gemm_f32.hpp"
 #include "cpu/gemm/s8x8s32/ref_gemm_s8x8s32.hpp"
 #include "cpu/gemm/s8x8s32/simple_gemm_s8s8s32.hpp"
+#ifdef DNNL_PPC64
+#include "cpu/ppc64/PPC64_gemm_driver.hpp"
+using namespace dnnl::impl::cpu;
+#endif
```

Comment:
Please move this under:
```
#if DNNL_X64
...
#elif DNNL_PPC64
...
#endif
```

Please also do not use CamelCase for file names.

Reply:
I looked in wikipedia for the definition of CamelCase, and I'm now confused.
Are you saying you prefer "ppc64_gemm_driver.hpp" over "PPC64_gemm_driver.hpp"?
If so, we can certainly do that.  I'm just trying to be sure I know what you're wanting.

Reply:
fixed CamelCase filenames and subroutine call name in fourth commit.